app.py

```diff
@@ -1853,6 +1853,14 @@ def tryit(level, program_id):
         ))
 
 
+@app.route('/search_students', methods=['GET'])
+def filter_usernames():
+    search = request.args.get('search', '')
+    results = DATABASE.get_users_that_starts_with(search)
```

Comment:
Probably you want to only do an actual search once you've gotten at least one character.

Otherwise the first query will always return `["000_test", "4242 i like kittens", "aaron a. aaronson", ...]`.

Reply:
Oh for sure! I was thinking about that yesterday, I'm only going to look data if I there's more than one character.